effort_controllers/test/simple_bot_goto_position.cpp

```diff
@@ -0,0 +1,70 @@
+#include <ros/ros.h>
```

Comment:
I added the license and I also updated the one of `simple_bot_hw.cpp`. They are now a copy of the one from `joint_position_controller.cpp` with my name added. I am never 100% sure about licenses, so feel free to correct me if something is wrong with this.

Reply:
I'm never sure either :)

@bmagyar look good?

Reply:
TBH I wouldn't be too fussed about putting license notes in every file, especially since the package.xml states BSD3. If you'd like though, that's ok but don't attribute new files to Willow Garage or HiDof, if it's just you, put yourself.